### DISCUSSION TITLE
Are VxWorks executables built as DKM or RTP?

### DISCUSSION THREAD

**Discussion by asvatos1** [nasa/cFS#370] [Q&A] [answered]
Hi,

I would like to know if VxWorks executable built by 'toolchain-ppc-vxworks6.9.cmake' is a Real Time Processes (RTP) or Downloadable Kernel Module (DKM). I am rolling my own toolchain file and it isn't obvious which type should be produced.

Warm regards,

Adam

**Comment by ghost** [comment, marked as answer]
Hi Adam,
The executable is a DKM. Work has been done to build and run cFS in an RTP, but it is not an official port yet.
Alan

**Reply by asvatos1** [reply]
Thank you, Alan. 👍

**Comment by dtran11** [comment]
Any updates on this development? Thanks
